src/components/shell-panel/interfaces.d.ts

```diff
@@ -0,0 +1 @@
+export type DisplayMode = "docked" | "detached" | "overlaid";
```

Comment:
WDYT about these not being in the past (`dock | detach | overlay`)? Not sure about `detach` though. Maybe `isolate` or `separate`?

Reply:
It seems to match other tenses (`escapeDisabled`, `textEnabled`) - and keeps the existing `detached` nomenclature. I think `isolate` or `separate` is... maybe less clear?

Reply:
> It seems to match other tenses (escapeDisabled, textEnabled)

Those are prop names and not the values. I don't think we have anything similar for existing modes. I could be wrong.

> and keeps the existing detached nomenclature. I think isolate or separate is... maybe less clear?

Nothing good ol' doc won't fix, although I do hear you on `isolate` and `separate`. Do you think `detach` would still be less clear?

Reply:
I agree since they are now values and not properties we should probably have them to not be past tense.

@macandcheese can you come up with the values? I think the only one in question is detach right? What about "float"?

Reply:
👍🏼  for `float`! 🎪🤡🎈